final/README.md

```diff
@@ -1,19 +1,36 @@
-# FINAL CODE - Odyssey Voyage II - Server (Airlock)
+# Odyssey Voyage II - Server (Airlock)
 
-This is the final state of the codebase after going through the Odyssey Voyage II course.
-
-You can [find the course lessons and instructions on Odyssey](http://odyssey.apollographql.com/voyage-part2), Apollo's learning platform.
+Welcome to the companion app of Odyssey's Voyage II: Federating the Monolith! This is the `server` backend of the Airlock app. You can [find the course lessons and instructions on Odyssey](http://apollographql.com/tutorials/voyage-part2), Apollo's learning platform.
 
 You can [preview the completed demo app here](https://odyssey-airlock.netlify.app/).
 
 You can [find the client counterpart here](https://github.com/apollographql/odyssey-voyage-II-client).
 
-## How to use this repo for the final version of the course project
+## How to use this repo
 
-The course will walk you through step by step how to turn this monolithic graph into a federated graph. This codebase is the final point of your journey!
+The course will walk you through step by step how to turn this monolithic graph into a federated graph. This codebase is the starting point of your journey!
 
 To get started:
 
+1. Run `npm install`.
+1. Run `npm start`.
+
+This will start the GraphQL API server on [http://localhost:4000](http://localhost:4000)
+
+Next, let's run some local services.
+
+1. In a new terminal window, run `npm run launch`. This will run 4 local services, which you can learn about in the accompanying Odyssey course.
```

Comment:
```suggestion
1. In a new terminal window, run `npm run launch`. This will run four local services, which you can learn about in the accompanying Odyssey course.
```